db/migrate/20240601204027_create_active_storage_tables.active_storage.rb

```diff
@@ -1,10 +1,7 @@
 # This migration comes from active_storage (originally 20170806125915)
 class CreateActiveStorageTables < ActiveRecord::Migration[7.0]
   def change
-    # Use Active Record's configured type for primary and foreign keys
-    primary_key_type, foreign_key_type = primary_and_foreign_key_types
-
-    create_table :active_storage_blobs, id: primary_key_type do |t|
+    create_table :active_storage_blobs, id: :uuid do |t|
```

Comment:
@ManuSG8 needed to support our models that are using `uuid` as a primary key type